core/src/Microsoft.Identity.Core.csproj

```diff
@@ -32,20 +32,23 @@
   <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard1.1' ">
     <DefineConstants>$(DefineConstants);FACADE</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp1.0' ">
+    <DefineConstants>$(DefineConstants);NET_CORE</DefineConstants>
+  </PropertyGroup>
   <PropertyGroup Condition="'$(TargetFramework)' == 'uap10.0' ">
-    <DefineConstants>$(DefineConstants);WINDOWS_APP</DefineConstants>
+    <DefineConstants>$(DefineConstants);WINDOWS_APP;SUPPORTS_UI</DefineConstants>
```

Comment:
I'm not using this SUPPORTS_UI yet, I plan to use it to throw nicer exception in case someone targets netstandard from other platforms, for example Windows Phone or PCL

Reply:
ack. consider though not adding it before using it.

---
In reply to: [208391266](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1175#discussion_r208391266) [](ancestors = 208391266)